src/jquery.formset.js

```diff
@@ -92,6 +92,7 @@
                         del.val('on');
                         row.hide();
                         forms = $('.' + options.formCssClass).not(':hidden');
+                        totalForms.val(forms.length);
```

Comment:
If I delete one item and create another in one POST, the new item is not created, only the deletion happens. It works fine if I remove this line.

Reply:
Thanks a lot, I couldn't upgrade my project because this was an issue with my formsets, but worked perfectly with your solution.